bioconda_utils/bioconda_utils-conda_build_config.yaml

```diff
@@ -2,6 +2,10 @@
 # in
 # https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/conda_build_config.yaml
 
+# clear hard-coded default value for CONDA_BUILD_SYSROOT
+CONDA_BUILD_SYSROOT:
+  - ''
```

Comment:
Setting this to an empty string works because of https://github.com/AnacondaRecipes/aggregate/blob/afa864187423917054a307ac4d9f04d3d2463fcc/clang/activate-clang.sh#L122 :
```sh
 "CONDA_BUILD_SYSROOT,${CONDA_BUILD_SYSROOT:-$(xcrun --show-sdk-path)}"
```
which sets resets the variable when it is set but empty.

Reply:
@mbargull Is that documented somewhere? Or is that undocumented feature? Should we have that noted in the yaml as a comment?